crates/bitwarden-wasm-internal/src/client.rs

```diff
@@ -27,7 +29,7 @@ fn convert_level(level: LogLevel) -> Level {
 // Rc<...> is to avoid needing to take ownership of the Client during our async run_command
 // function https://github.com/rustwasm/wasm-bindgen/issues/2195#issuecomment-799588401
 #[wasm_bindgen]
-pub struct BitwardenClient(Rc<Client>);
+pub struct BitwardenClient(pub(crate) Rc<Client>);
```

Comment:
```suggestion
pub struct BitwardenClient(Rc<Client>);
```

Nit: This inner value is only used on it's own impl as far as I know, so we don't need it to be `pub(crate)`